.devcontainer/cuda12.9-conda/devcontainer.json

```diff
@@ -11,7 +11,9 @@
   "runArgs": [
     "--rm",
     "--name",
-    "${localEnv:USER:anon}-rapids-${localWorkspaceFolderBasename}-25.10-cuda12.9-conda"
+    "${localEnv:USER:anon}-rapids-${localWorkspaceFolderBasename}-25.10-cuda12.9-conda",
+    "--ulimit",
+    "nofile=500000"
```

Comment:
It might be worth replicating options we use in CI: https://github.com/rapidsai/rapidsmpf/blob/2de638653ebaf4882e9e1f4d9a1d5eaa571b80f4/.github/workflows/pr.yaml#L98, those are primarily important for UCX.